rootfs/etc/cont-init.d/03-config.sh

```diff
@@ -181,11 +181,12 @@ cat > /var/www/rutorrent/conf/config.php <<EOL
 \$XMLRPCMountPoint = "/RPC2"; // DO NOT DELETE THIS LINE!!! DO NOT COMMENT THIS LINE!!!
 
 \$pathToExternals = array(
-    "php"   => '',
-    "curl"  => '',
-    "gzip"  => '',
-    "id"    => '',
-    "stat"  => '',
+    "php"    => '',
+    "curl"   => '',
+    "gzip"   => '',
+    "id"     => '',
+    "stat"   => '',
+    "python" => '/usr/bin/python3',
```

Comment:
Would be better to use `"python" => '$(which python3)',` to solve path.

Reply:
Changing this now 👍